### ISSUE TITLE
projectile-multi-term-in-root is not defined until multi-term has been opened once

### ISSUE THREAD

**Issue by brandoncc** [syl20bnr/spacemacs#6162]
#### Description
`projectile-multi-term-in-root` is not defined until multi-term has been opened once.

#### Reproduction guide
- Start Emacs
- Try to execute `projectile-multi-term-in-root` (either using `M-x` or `SPC p $ t`).
- Open multi-term
- Try again, this time the function and keybinding will be available

*Observed behaviour:*
`projectile-multi-term-in-root` is not defined until multi-term has been opened once because the function is defined in the `config` block of `use-package`.

*Expected behaviour:*
`projectile-multi-term-in-root` should be available when you open Emacs

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.20
- Spacemacs branch: master (rev. 9c7250a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion better-defaults emacs-lisp git markdown org
                 (shell :variables shell-default-height 30 shell-default-position 'bottom shell-default-shell 'eshell)
                 spell-checking syntax-checking version-control ruby ruby-on-rails osx github themes-megapack html javascript eyebrowse yaml dash ranger vinegar colors seeing-is-believing gtags erc quickrun php ibuffer emoji browserify clone-github-repo check-acronym)
```

**Comment by TheBB** [comment]
Fixed in https://github.com/syl20bnr/spacemacs/pull/6169

However, do we really need this when we have `SPC p '`? If so we should extend to the other shell types as well.

**Comment by brandoncc** [comment]
I can only speak for myself, but I asked for two weeks in the gitter room how I could make multi term open in my project directory before I finally stumbled across this yesterday. Also, `SPC p '` is not defined for me, I'm not sure why that would be...maybe a similar bug? I only have `SPC '`.

**Comment by brandoncc** [comment]
cc @TheBB

**Comment by TheBB** [comment]
Maybe it's only on develop?

No need to CC me by the way, I'm already participating in this thread.

**Comment by brandoncc** [comment]
Ah, sorry about that. If `SPC p '` does the same thing I am perfectly fine with this being removed. I actually like that binding more anyway. I can wait until it makes it to master :smiley: I spent some time learning how to use eshell yesterday which is the only shell that maintains the current directory by default (I assume because of its elisp roots), so I can survive now.

Thanks for the fix and the info 👍

**Comment by syl20bnr** [comment]
Should be fixed in develop thanks to @TheBB :-)

**Comment by brandoncc** [comment]
Thank you guys! 👍

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
